pdns/dnsdist.cc

```diff
@@ -2390,6 +2390,32 @@ static void setupLocalSocket(ClientState& clientState, const ComboAddress& addr,
         warnlog(e.what());
       }
     }
+#ifdef __linux__
```

Comment:
I don't think we want to raise automatically if either `g_socketUDPSendBuffer` or `g_socketUDPRecvBuffer` is set.